static/docs.css

```diff
@@ -108,7 +108,7 @@ a.sc-logo img {
   font-size: 16px;
 }
 
-.side-nav .nav-header a, .side-nav .nav-header span {
```

Comment:
The second span is used to replace the normal title with a link if javascript is enabled. Disable javascript, and this will probably break it.